build/write-image.sh

```diff
@@ -21,10 +21,12 @@ sudo e2label ${OUTPUT_DEVICE}p4 blue
 mkdir -p /tmp/eos-mnt
 sudo mount ${OUTPUT_DEVICE}p1 /tmp/eos-mnt
 
+sudo sed -i 's/^/modprobe.blacklist=uas /g' /tmp/eos-mnt/cmdline.txt
 sudo sed -i 's/LABEL=writable/LABEL=green/g' /tmp/eos-mnt/cmdline.txt
 cat /tmp/eos-mnt/config.txt | grep -v "dtoverlay=" | sudo tee /tmp/eos-mnt/config.txt.tmp
 echo "dtoverlay=pwm-2chan" | sudo tee -a /tmp/eos-mnt/config.txt.tmp
 sudo mv /tmp/eos-mnt/config.txt.tmp /tmp/eos-mnt/config.txt
+sudo cp /tmp/eos-mnt/config.txt /tmp/eos-mnt/config.txt.orig
```

Comment:
document the purpose of this and how it ties into the overall PR. Or is it just cleanup you noticed?